research/picai/fl_nnunet/nnunet_client.py

```diff
@@ -51,6 +52,14 @@
     from nnunetv2.training.nnUNetTrainer.nnUNetTrainer import nnUNetTrainer
     from nnunetv2.utilities.dataset_name_id_conversion import convert_id_to_dataset_name
 
+log(
+    WARNING,
+    (
+        "You are using the old version of nnUNetClient from the research folder. "
```

Comment:
It is not a big deal either way but we can always use the commit history to revert or reference an old version of the client or server

Reply:
Let's leave it for now and just delete it in a subsequent PR